src/java.base/share/classes/java/util/Scanner.java

```diff
@@ -78,7 +78,16 @@
  *     }
  * }
  *
- * <p>As another example, this code allows {@code long} types to be
+ * <p>This code reads lines from {@link System#in} with {@link
+ * System##stdin.encoding stdin.encoding}:
```

Comment:
"from System.in with stdin.encoding" isn't very clear. I think you we can expand the sentence, or make it two sentences, so to that is uses a Scanner to read lines from System.in, and uses the value of stdin.encoding as the charset.

Reply:
Made it two sentences.